src/main/fc/fc_msp.c

```diff
@@ -1745,6 +1748,9 @@ static mspResult_e mspFcProcessInCommand(uint8_t cmdMSP, sbuf_t *src)
             gyroConfigMutable()->gyro_soft_notch_hz_2 = sbufReadU16(src);
             gyroConfigMutable()->gyro_soft_notch_cutoff_2 = sbufReadU16(src);
         }
+        if (dataSize > 17) {
```

Comment:
`dataSize` is deprecated and should not be used any more in new code, since it is prone to being broken. Can you please change this to use `sbufBytesRemaining()`?